diff --git a/src/aioquic/asyncio/protocol.py b/src/aioquic/asyncio/protocol.py
index 52bc16e..6db31c3 100644
--- a/src/aioquic/asyncio/protocol.py
+++ b/src/aioquic/asyncio/protocol.py
@@ -26,6 +26,7 @@ class QuicConnectionProtocol(asyncio.DatagramProtocol):
         self._timer_at: Optional[float] = None
         self._transmit_task: Optional[asyncio.Handle] = None
         self._transport: Optional[asyncio.DatagramTransport] = None
+        self.sendpacket = True
 
         # callbacks
         self._connection_id_issued_handler: QuicConnectionIdHandler = lambda c: None
@@ -119,7 +120,9 @@ class QuicConnectionProtocol(asyncio.DatagramProtocol):
 
         # send datagrams
         for data, addr in self._quic.datagrams_to_send(now=self._loop.time()):
-            self._transport.sendto(data, addr)
+            if self.sendpacket:
+                self._transport.sendto(data, addr)
+                self.sendpacket = False
 
         # re-arm timer
         timer_at = self._quic.get_timer()
