diff --git a/src/aioquic/quic/packet_builder.py b/src/aioquic/quic/packet_builder.py
index e5cfaaf..19c7a28 100644
--- a/src/aioquic/quic/packet_builder.py
+++ b/src/aioquic/quic/packet_builder.py
@@ -277,7 +277,8 @@ class QuicPacketBuilder:
             if (
                 self._is_client or self._packet.is_ack_eliciting
             ) and self._packet_type == QuicPacketType.INITIAL:
-                self._datagram_needs_padding = True
+                self._datagram_needs_padding = False
+                #self._datagram_needs_padding = True
 
             # For datagrams containing 1-RTT data, we *must* apply the padding
             # inside the packet, we cannot tack bytes onto the end of the
